frontend/src/components/form/base/__tests__/EDatePicker.spec.js

```diff
@@ -0,0 +1,122 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+
+import i18n from '@/plugins/i18n'
+import formBaseComponents from '@/plugins/formBaseComponents'
+import { vueMoment, moment } from '@/plugins/vueMoment'
+
+import { mount as mountComponent } from '@vue/test-utils'
+import EDatePicker from '../EDatePicker'
+
+Vue.use(Vuetify)
+Vue.use(formBaseComponents)
+Vue.use(vueMoment, { moment })
+
+describe('An EDatePicker', () => {
+  let vuetify
+
+  const DATE_1 = '2020-03-01'
+  const DATE_2 = '2015-12-31'
+  const INVALID_DATE_1 = 'some date'
+  const INVALID_DATE_2 = '2026-13-01'
+
+  const localeData = [
+    [
+      'de',
+      {
+        date_1: '01.03.2020',
+        date_2: '31.12.2015',
+        date_3: '24.03.2020'
+      }
+
+    ],
+    [
+      'en',
+      {
+        date_1: '03/01/2020',
+        date_2: '12/31/2015',
+        date_3: '03/24/2020'
+      }
+
+    ]
+  ]
+
+  const flushPromises = () => new Promise((resolve) => setImmediate(resolve))
+
+  const waitForDebounce = () => new Promise((resolve) => setTimeout(resolve, 110))
```

Comment:
How sure are you, that the 110 ms are always enough?

Reply:
Pretty sure: https://github.com/ecamp/ecamp3/blob/3d597fa33ac5264f51fd01253bf4223d5d6de320/frontend/__mocks__/lodash.js#L9

As stated in https://github.com/ecamp/ecamp3/pull/706#discussion_r541788463 I'd actually like to get rid of it for the tests, or at least decrease the timeout to some smaller value. But I don't know why it was needed in the first place. Maybe @usu you have some context here?